src/board.hpp

```diff
@@ -3,13 +3,25 @@
 
 #include "coords.hpp"
 #include "enums.hpp"
+#include <vector>
+#include <optional>
+#include <array>
+
 
 namespace TicTacToe {
+    using BoardLine = std::array<Coords, BOARD_WIDTH>;
     class Board {
+    private:
+        std::optional <Symbol> board [BOARD_WIDTH][BOARD_WIDTH];
+        ;
     public:
-        std::optional <Symbol> fieldContent(Coords coords);
+        std::vector<BoardLine> winningPossibilities();
+
+        std::vector<BoardLine> testTaMetoda();
+        std::optional <Symbol> fieldContent(Coords fieldCoords);
+        void changeFieldContent(Coords fieldCoords, Symbol symbol);
+//        std::vector<Coords[3]> winningSymbolLines();
```

Comment:
don't commit commented code if it doesn't add to code readability

for real, using some additional util for committing makes partial commits so much less cumbersome and lets you see what you're actually committing